plugins/woocommerce-blocks/assets/js/blocks/collection-filters/inner-blocks/rating-filter/block.json

```diff
@@ -0,0 +1,44 @@
+{
+	"name": "woocommerce/collection-rating-filter",
+	"version": "1.0.0",
+	"title": "Collection Rating Filter",
+	"description": "Enable customers to filter the product collection by rating.",
+	"category": "woocommerce",
+	"keywords": [ "WooCommerce" ],
+	"supports": {
+		"interactivity": true
+	},
+	"ancestor": [ "woocommerce/collection-filters" ],
+	"usesContext": [ "collectionData" ],
+	"attributes": {
+		"className": {
+			"type": "string",
+			"default": ""
+		},
+		"showCounts": {
+			"type": "boolean",
+			"default": false
+		},
+		"displayStyle": {
+			"type": "string",
+			"default": "dropdown"
```

Comment:
The default style of rating filter block is dropdown, but others are checkbox lists. I think we should make it consistent. Either change this to checkbox list or others to dropdown are good to me.

Reply:
Yep this is by accident, I'll fix.